luigi_pipeline/seqr_gcnv_loading.py

```diff
@@ -13,6 +13,21 @@
 
 SAMPLE_ID_REGEX = r'(.+)_v\d+_Exome_(C|RP-)\d+$'
 
+EXPECTED_TYPES = {
```

Comment:
The `type` parameter of the `import_table` is a field type assignment. So you may want to change it to something like `FIELD_TYPES`.